### PR TITLE
HOTFIX: Fix Security Shell Gun being uncraftable.

### PR DESCRIPTION
**Pull request by BramvanZijp**
## About the PR
Add the sec shell gun to the lathe list since I forgor.

## Why / Balance
Because I forgor and it should already be there.

## Technical details
No

## Media
No

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl: BramvanZijp
- fix: The Security Shell Gun can now actually be crafted.

### CODE REVIEW

**Review by Errant-4 — APPROVED** [review]
This was mentioned as being an intended part of #32829, so hotfix is warranted. I tested and it works

**Review by slarticodefast — APPROVED** [review]

**Comment by chromiumboy** [comment]
Thanks for this. I see that this has two other maintainer approvals, so its good to go

**Pull request merged**
